x-pack/libbeat/autodiscover/providers/aws/elb/_meta/fields.yml

```diff
@@ -0,0 +1,84 @@
+- key: elb_listener
+  title: "ELB Listener"
+  description: >
+    ELB Listeners
+  short_config: false
+  release: experimental
+  fields:
+    - name: elb_listener
+      type: group
+      description: >
+          jolokia contains metrics exposed via jolokia agent
+      fields:
+        - name: ip_address_type
+          type: keyword
+          level: core
```

Comment:
Where is this level part come from? We only have this in ECS AFAIK.

Reply:
Ohhhh, I was just blindly copy pasting ;)